### DISCUSSION TITLE
Nemo - Limiting file emblem size to a max pixel size

### DISCUSSION THREAD

**Discussion by mindinsomnia** [linuxmint/discussions#200] [Development]
I find the way Nemo renders emblems over file icons to be a tad annoying. Because the emblems are very large, and for images particularly, the file emblems can sometimes cover up to half of an image's thumbnail (for example a landscape ratio image).

I'm trying to find where in the code I need to go to modify this behaviour, in the hope of making a pull request to offer, but I can't figure out where exactly in the codebase to go to modify this behaviour. Does anyone have any advice?

**Comment by Jeremy7701** [comment]
I'd suggest that https://github.com/linuxmint/nemo-extensions/blob/master/nemo-emblems/nemo-extension/nemo-emblems.py might be worth a look.
